src/components/User.spec.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react'
+import User from './User'
+
+const data = require('../data/data.json')
```

Comment:
the data is hard coded in your component. I think there is no need to import them again.